crates/db_views/src/comment_view.rs

```diff
@@ -115,6 +124,7 @@ fn queries<'a>() -> Queries<
       .field(community_moderator::community_id)
       .nullable()
       .is_not_null(),
+    local_user::admin.nullable().is_not_null(),
```

Comment:
based on the changes @dullbananas has been doing for PostView (removing left joins), this would probably be better as a `exists(local_user::table.find(comment::creator_id).filter(local_user::admin))`

Reply:
That one didn't work, but I was able to refactor the joins into subqueries.